### DISCUSSION TITLE
Flip FFC connector 180°?

### DISCUSSION THREAD

**Discussion by tobymurray** [vroland/epdiy#363] [Q&A] [answered]
My ED060XD4 with the v7 EPDiy ends up flipped from how it feels like it _should_ be (so the board could tuck behind the screen.

![image](https://github.com/user-attachments/assets/072a163e-327d-49ad-9f69-74e764c9386b)

Is there an obvious general solution to this that I can buy off the shelf? It feels like this would be a part that exists, but I'm not finding specific results (I also have very little idea what I'm doing, so there's that).

I've tried searching for e.g. "34-pin FFC adapter 180-degree flip". Mouser and Digikey appear to have many thousands of offerings that _aren't_ what I'm looking for.

**Comment by martinberlin** [comment, marked as answer]
There is no a real solution for that unless you:

- Use an extensor cable and then you place it whenever you want (And maybe make a case for it)
- Make a specially modified PCB where you place the connector where you need it. You already had success making the PCB so why not?

What exist in Aliexpress are cables that have FPC connectors on top, and in the other end at the bottom, so with that and an adapter you can somehow rotate it (Called "reverse direction"). But take care because you need to know what you are doing or you can invert all the pins, effectively sending voltages or data where they should not go.
If you invert it, make sure to check with a multimeter, that the high voltage pins correspond at least on the corners to what the display expects (Check schematics)
